service/frontend/workflowHandler.go

```diff
@@ -87,6 +87,7 @@ type (
 		PersistenceToken  []byte
 		TransientDecision *gen.TransientDecisionInfo
 		EventStoreVersion int32
+		CreateTaskID      int64
```

Comment:
Is this backward compatible?

Reply:
public API does not use this field,
only re-send stack use it